Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -29,7 +29,8 @@ IndicatorInfoProvider {
     private let type: RepositoryIssuesType
     private let searchKey: ListDiffable = "searchKey" as ListDiffable
     private let debouncer = Debouncer()
-    private var previousSearchString = "is:open "
+    private var previousSearchString = ""
+    private var prefix = "is:open "
```

Comment:
Why does the GitHub highlighter think `prefix` is a keyword? 🤔

Reply:
:shrug: Problem?

Reply:
No should be good 😊